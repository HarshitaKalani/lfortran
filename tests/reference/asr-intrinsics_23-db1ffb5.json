{
    "basename": "asr-intrinsics_23-db1ffb5",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_23.f90",
    "infile_hash": "3e162c296e5df394ddaea3689028b9cb97cd97e8bf9fdc0296eb6697",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_23-db1ffb5.stdout",
    "stdout_hash": "8bed8a2f63f6031ef880b817fd7369125eea85c02dc296b891515cf4",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}