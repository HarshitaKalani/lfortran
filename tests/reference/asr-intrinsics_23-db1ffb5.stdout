(TranslationUnit
    (SymbolTable
        1
        {
            intrinsics_23:
                (Program
                    (SymbolTable
                        2
                        {
                            i32:
                                (Variable
                                    2
                                    i32
                                    []
                                    Local
                                    (IntrinsicScalarFunction
                                        Huge
                                        [(IntegerConstant 0 (Integer 4))]
                                        0
                                        (Integer 4)
                                        (IntegerConstant 2147483647 (Integer 4))
                                    )
                                    (IntegerConstant 2147483647 (Integer 4))
                                    Parameter
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            i64:
                                (Variable
                                    2
                                    i64
                                    []
                                    Local
                                    (IntrinsicScalarFunction
                                        Huge
                                        [(IntegerConstant 0 (Integer 8))]
                                        0
                                        (Integer 8)
                                        (IntegerConstant 9223372036854775807 (Integer 8))
                                    )
                                    (IntegerConstant 9223372036854775807 (Integer 8))
                                    Parameter
                                    (Integer 8)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            y:
                                (Variable
                                    2
                                    y
                                    []
                                    Local
                                    (IntrinsicScalarFunction
                                        Huge
                                        [(RealConstant
                                            0.000000
                                            (Real 4)
                                        )]
                                        0
                                        (Real 4)
                                        (RealConstant
                                            340282346638528859811704183484516925440.000000
                                            (Real 4)
                                        )
                                    )
                                    (RealConstant
                                        340282346638528859811704183484516925440.000000
                                        (Real 4)
                                    )
                                    Parameter
                                    (Real 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            z:
                                (Variable
                                    2
                                    z
                                    []
                                    Local
                                    (IntrinsicScalarFunction
                                        Huge
                                        [(RealConstant
                                            0.000000
                                            (Real 8)
                                        )]
                                        0
                                        (Real 8)
                                        (RealConstant
                                            179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000
                                            (Real 8)
                                        )
                                    )
                                    (RealConstant
                                        179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000
                                        (Real 8)
                                    )
                                    Parameter
                                    (Real 8)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    intrinsics_23
                    []
                    [(Print
                        [(Var 2 i32)
                        (Var 2 i64)
                        (Var 2 y)
                        (Var 2 z)]
                        ()
                        ()
                    )
                    (Print
                        [(IntrinsicScalarFunction
                            Huge
                            [(IntegerConstant 0 (Integer 4))]
                            0
                            (Integer 4)
                            (IntegerConstant 2147483647 (Integer 4))
                        )
                        (IntrinsicScalarFunction
                            Huge
                            [(IntegerConstant 0 (Integer 8))]
                            0
                            (Integer 8)
                            (IntegerConstant 9223372036854775807 (Integer 8))
                        )
                        (IntrinsicScalarFunction
                            Huge
                            [(RealConstant
                                0.000000
                                (Real 4)
                            )]
                            0
                            (Real 4)
                            (RealConstant
                                340282346638528859811704183484516925440.000000
                                (Real 4)
                            )
                        )
                        (IntrinsicScalarFunction
                            Huge
                            [(RealConstant
                                0.000000
                                (Real 8)
                            )]
                            0
                            (Real 8)
                            (RealConstant
                                179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000
                                (Real 8)
                            )
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
